sessions/databases/time-to-optimize-poorly-optimized.md

```diff
@@ -7,9 +7,9 @@ talk_tags: [communication, databases, story]
 session_slug: databases
 # ---- Edit information below this line ----
 # The title of your talk
-talk_title: "No Time to Optimize: A Poorly Optimized Query Story"
+talk_title: "Exploring Query Optimization: How a few lines of code can save hours of time"
 # A short version of the title, suitable for small displays
-talk_title_short: No Time to Optimize
+talk_title_short: Exploring Query Optimization
 # A link to your talk's materials, when ready
 talk_materials_url: ~
```

Comment:
Just a quick note that if you put your slides online, you can come back and add the URL in this line (it's a big help to people who benefit from reading the slides while listening to your talk).